lib/ansible/plugins/lookup/consul_kv.py

```diff
@@ -53,6 +53,10 @@
   - name: retrieving a KV from a remote cluster on non default port
     debug:
       msg: "{{ lookup('consul_kv', 'my/key', host='10.10.10.10', port='2000') }}"
+
+  - name: retrieving a KV from a remote cluster on non default port with ACL enabled
+    debug:
+      msg: "{{ lookup('consul_kv', 'my/key', host='10.10.10.10 token=a20f2009-a1cf-4e19-8286-0632ea3bcad6 recurse=yes', port='2000') }}"
```

Comment:
This looks odd - shouldn't each parameter be separated by a `,` and have its own `'` around it? I can't test this, but I would guess it should be:

```suggestion
      msg: "{{ lookup('consul_kv', 'my/key', host='10.10.10.10', token='a20f2009-a1cf-4e19-8286-0632ea3bcad6', recurse='yes', port='2000') }}"
```

Reply:
No, that's the tricky part. There was no documentation about this so after several tests and reading source code I realised the token and the recurse attributes are separated by a blank space

Reply:
Since I put that comment on this PR, I've run into this same construction elsewhere - see https://github.com/ansible/ansible/pull/46423#issuecomment-488168912. This isn't the modern way to do this, but it does happen in more than one place in the Ansible source code.